tests/src/main/java/com/hazelcast/simulator/tests/map/IntByteMapTest.java

```diff
@@ -58,6 +59,7 @@
 
     private IMap<Integer, Object> map;
     private int[] keys;
+    private byte[][] values;
```

Comment:
not correct.

The values are not the data-set. the key count determines the data-set. With 1 single value of 1kb you can still create a data-set of key-count = 1M.